amplify/backend/function/PropertyFunction/src/index.py

```diff
@@ -1,40 +1,89 @@
 import json
 import mysql.connector
 
+import boto3
+from botocore.exceptions import ClientError
 
-try:
-  db=mysql.connector.connect(
-    host="tectown-backend-q1-2024.c1s0muoa0qc4.us-east-1.rds.amazonaws.com",
-    user="admin",
-    database="rentalHouses",
-    password="Tectown1!"
-  )
-  print("database connected")
-except Exception as e:
-  print(f'There was an exception: {e}')
+
+def get_secret():
+    secret_name = "dev/rentalHouseApp"
+    region_name = "us-east-1"
+
+    # Create a Secrets Manager client
+    session = boto3.session.Session()
+    client = session.client(
+        service_name='secretsmanager',
+        region_name=region_name
+    )
+
+    try:
+        get_secret_value_response = client.get_secret_value(
+            SecretId=secret_name
+        )
+    except ClientError as e:
+        raise e
+
+    secret = get_secret_value_response['SecretString']
+    return json.loads(secret)
+
+
+    # Your code goes here.
+def connect_to_database():
+    # Fetch secrets from AWS Secrets Manager
+    secrets = get_secret()
```

Comment:
We should enclose the get_secret() function in a try-except block because there's a possibility it will raise an error (In the definition of the function, we're not handling the exception there. Instead we're raising it and sending it to the parent function to be handled there)